### ISSUE TITLE
More project colors

### ISSUE THREAD

**Issue by vanderlee** [toggl/toggldesktop#1916]
The current selection of possible project colors lacks some variety.
I understand the preference for earthtones, but that doesn't explain the lack of shades of yellow, light gray and a good medium green.

**Comment by IndrekV** [comment]
Hi, we have plans to improve the color selection in Toggl overall. Once this is done more colors will be also added to desktop apps. Maybe even a color picker to allow users to pick what ever they want.

**Comment by IndrekV** [comment]
Latest news on the colors is that the selection colors will be updated (same as in the web version) and color picker will be added. There is a separate issue for that #2008
